db/tweets.sql

```diff
@@ -0,0 +1,42 @@
+-- USERDATA table for holding the user information (USERNAME, user full name, and user ID)
+CREATE TABLE USERDATA(
+    USER_ID NUMBER NOT NULL,
+    USERNAME VARCHAR2(255 char) NOT NULL,
+    USERFULLNAME VARCHAR2(255 char),
+    PRIMARY KEY (USER_ID)
+);
+
+COMMENT ON TABLE USERDATA IS 'Contains the data for the user, e.g (USER ID, USERNAME, USER FULL NAME)';
+COMMENT ON COLUMN USERDATA.USER_ID IS 'ID OF users';
+COMMENT ON COLUMN USERDATA.USERNAME IS 'Username name';
+COMMENT ON COLUMN USERDATA.USERFULLNAME IS 'User full name';
+
+-- sequence counter for the unique user ID, starts from 1 and increment by 1
+CREATE SEQUENCE seq_USER_ID
+MINVALUE 1
+START WITH 1
+INCREMENT BY 1
+CACHE 10;
+
+--INSERT INTO USERDATA (USER_ID)
+--VALUES (seq_USER_ID.nextval);
+
+-- TWEET_DATA table holds the tweet information, (message, tweet ID, TITLE, mode)
+CREATE TABLE TWEET_DATA(
+    USER_ID NUMBER NOT NULL,
+    ID  VARCHAR2(255 char)  NOT NULL,
```

Comment:
Instead of using generic `ID`, can you name it explicitly like `TWEET_ID` to avoid confusion?